### DISCUSSION TITLE
Results based on item permission

### DISCUSSION THREAD

**Discussion by an3dree** [microsoft-search/pnp-modern-search#2014] [Q&A] [answered]
Hello everyone, and thanks a lot for this impressive work on modern search web parts

I have a question,

can i filter the results based on permission 

eg: just show a result if the user is RestrictedViwer on item permision level

best regards and thanks

**Comment by cleberto** [comment]
Up

**Comment by FranckyC** [comment, marked as answer]
Hi @Aan3dree, unfortunately, I don't think this is something possible as this information in not available as refiners and search results are already security trimmed. The only way to do this would be to set this information manually at item level using a custom column + managed property + a sync process with current permissions. The needed amount of work to do that is not worth it in my opinion.

**Reply by an3dree** [reply]
Hi @FranckyC , thanks for the quicky reply, it's now clearer to me.
